include/ebpf_api.h

```diff
@@ -462,6 +483,17 @@ extern "C"
     void
     ebpf_link_close(_Frees_ptr_ struct bpf_link* link) EBPF_NO_EXCEPT;
 
+    /**
+     * @brief Free bpf_link structure without cleaning up the underlying fd.
+     *
+     * The file descriptor must be closed using \ref ebpf_close_fd.
+     *
+     * @param[in] link Pointer to the bpf_link structure.
+     * @retval The file descriptor of the link.
```

Comment:
```suggestion
     * @returns The file descriptor of the link.
```
Use `retval` only for listing specific return values since doxygen generates a 2 column table if you use retval.